### DISCUSSION TITLE
v0.28.0-rc1

### DISCUSSION THREAD

**Discussion by borg323** [LeelaChessZero/lc0#1579] [General]
* Multigather is now made the default (and also improved). Some search settings
  have changed meaning, so if you have modified values please discard them.
  Specifically, `max-collision-events`, `max-collision-visits` and
 `max-out-of-order-evals-factor` have changed default values, but other options
  also affect the search. Similarly, check that your gui is not caching the old
  values.
* Performance improvements for the cuda/cudnn backends.
* Support for policy focus during training.
* Larger/stronger 15b default net for all packages except android, blas and dnnl
  that get a new 10b network.
* The distributed binaries come with the mimalloc memory allocator for better
  performance when a large tree has to be destroyed (e.g. after an unexpected
  move).
* The `legacy` time manager will use more time for the first move after a long
  book line.
* The `--preload` command line flag will initialize the backend and load the
  network during startup.
* A 'fen' command was added as a UCI extension to print the current position.
* Experimental onednn backend for recent intel cpus and gpus.

<hr /><em>This discussion was created from the release <a href='https://github.com/LeelaChessZero/lc0/releases/tag/v0.28.0-rc1'>v0.28.0-rc1</a>.</em>

**Comment by Kovax007** [comment]
Policy focus! 😎

**Reply by vibranium21** [reply]
V28.0 great job!

**Comment by fsmosca** [comment]
The lc0 in `lc0-v0.28.0-rc1-windows-gpu-nvidia-cudnn.zip` crashes in gtx 1650s. What could be the issue?

### Log
```
...
0621 11:17:28.062433 13696 c:\projects\lc0\src\chess\uciloop.cc:225] << option name TimeManager type string default smooth
0621 11:17:28.062900 13696 c:\projects\lc0\src\chess\uciloop.cc:225] << option name LogFile type string default 
0621 11:17:28.063341 13696 c:\projects\lc0\src\chess\uciloop.cc:225] << uciok
0621 11:17:44.279036 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> setoption name LogFile value log.txt
0621 11:17:47.122987 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> isready
0621 11:17:47.123067 13696 c:\projects\lc0\src\chess\uciloop.cc:225] << readyok
0621 11:17:51.585679 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> ucinewgame
0621 11:17:51.594546 13696 c:\projects\lc0\src\neural\loader.cc:238] Found pb network file: D:\Chess\Engines\Lc0\lc0-v0.28.0-rc1-windows-gpu-nvidia-cudnn/751390.pb.gz
0621 11:17:51.790088 13696 c:\projects\lc0\src\neural\factory.cc:91] Creating backend [cudnn-auto]...
0621 11:17:51.842055 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:1046] Switching to [cudnn-fp16]...
0621 11:17:51.907766 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:920] CUDA Runtime version: 10.0.0
0621 11:17:51.908464 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:933] Cudnn version: 7.4.2
0621 11:17:51.909188 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:943] Latest version of CUDA supported by the driver: 11.3.0
0621 11:17:51.910107 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:951] GPU: NVIDIA GeForce GTX 1650 SUPER
0621 11:17:51.910719 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:952] GPU memory: 4 GiB
0621 11:17:51.911221 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:954] GPU clock frequency: 1770 MHz
0621 11:17:51.911749 13696 c:\projects\lc0\src\neural\cuda\network_cudnn.cc:955] GPU compute capability: 7.5
0621 11:18:23.655658 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> isready
0621 11:18:23.655754 13696 c:\projects\lc0\src\chess\uciloop.cc:225] << readyok
0621 11:18:30.704183 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> position startpos
0621 11:18:41.330248 13696 c:\projects\lc0\src\chess\uciloop.cc:136] >> go movetime 30000
0621 11:18:41.330527 13696 c:\projects\lc0\src\engine.cc:302] Timer started at 0621 11:18:30.714977
0621 11:18:41.330908 7688 c:\projects\lc0\src\mcts\search.cc:892] Start a watchdog thread.
0621 11:18:41.331785 13696 c:\projects\lc0\src\mcts\search.cc:816] Search started. 10616ms already passed.
0621 11:18:41.332640 18868 c:\projects\lc0\src\mcts\search.h:243] Started search thread.
0621 11:18:41.332769 20128 c:\projects\lc0\src\mcts\search.h:243] Started search thread.
```

**Reply by mooskagh** [reply]
It's a known problem which is fixed in master (the fix will appear in the release).
Pass `--backend-options=custom_winograd=false` to the command line for now as a workaround.
(or through UCI `Backend options`=`custom_winograd=false`).

**Reply by GosseRomkes** [reply]
Does somebody know when the new master will come as a release?

Gosse
